docs/ContribOperators.md

```diff
@@ -2455,9 +2455,11 @@ This version of the operator has been available since version 1 of the 'com.micr
 
   Group Query Self/Cross Attention.
   
-  Supports different number of heads for q and kv. Only supports causal or local attention.
-  Supports rotary position embedding.
-  Supports k-v cache.
+  *Highly recommend using k-v cache share buffer for both CPU and CUDA. Enabled through IOBinding past and present kv.
```

Comment:
Might not need add EP limitations to operator spec. 

The spec could be shared by other EP (like DirectML, QNN etc).